tests/lsp/depB-compiler-test.toit

```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2019 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+import .lsp-client show LspClient run-client-test
+import expect show *
+import system
+import system show platform
+
+main args:
+  run-client-test args: test it
+  // run-client-test --use-toitlsp args: test it
```

Comment:
In or out?

Reply:
For now: in. Will be removed in a different PR.
done.